app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ExpensesController < ApplicationController
+  def new
+    @expense = Expense.new
+  end
+
+  def create
+    @expense = Expense.new(expense_params)
+    if @expense.save
+      redirect_to root_path
+      flash[:success] = "Expense was successfully created"
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.